src/data/store.js

```diff
@@ -1,8 +1,8 @@
-import { configureStore } from '@reduxjs/toolkit';
-import examReducer from './slice';
-
-export default configureStore({
-  reducer: {
-    specialExams: examReducer,
-  },
-});
+// import { configureStore } from '@reduxjs/toolkit';
```

Comment:
I think we're okay to remove this, since we're still on a feature branch, right? Or were you thinking of doing that in a follow up pull request?

Reply:
Let me try removing that as part of this one

Reply:
Removing the store requires quite a bit of refactoring, because most of the tests directly import the store. I'll work through that now though!

Reply:
Oof! I didn't realize that. If you want to leave that to another ticket, this pull request looks good to merge to me, but I'm happy to re-review if you want to refactor the tests.

Reply:
It wasn't as bad as I expected because I could just use the test store as a replacement! Thankful that it existed. Should be set for another review now :)